pages/apis/rest_api/pipeline_templates.md

```diff
@@ -0,0 +1,293 @@
+# Pipeline templates API
+
+> 📘 Enterprise feature
+> Pipeline templates are only available on an [Enterprise](https://buildkite.com/pricing) plan.
+
+The pipeline templates API allows admins to create and manage pipeline templates for an organization.
+Non-admins will only be permitted to read pipeline templates marked as available by organization admins.
+
+## Pipeline template data model
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>uuid</code></th><td>UUID of the pipeline template</td></tr>
+  <tr><th><code>graphql_id</code></th><td><a href="/docs/apis/graphql-api#graphql-ids">GraphQL ID of the pipeline template</a></td></tr>
+  <tr><th><code>name</code></th><td>Name of the pipeline template</td></tr>
+  <tr><th><code>description</code></th><td>Description of the pipeline template</td></tr>
+  <tr><th><code>configuration</code></th><td>YAML step configuration for the pipeline template</td></tr>
+  <tr><th><code>available</code></th><td>When set to true, the pipeline template can assigned to pipelines by non-admins<br><em>Default:</em> <code>false</code></td></tr>
+  <tr><th><code>url</code></th><td>Canonical API URL of the pipeline template</td></tr>
+  <tr><th><code>web_url</code></th><td>URL of the pipeline template on Buildkite</td></tr>
+  <tr><th><code>created_at</code></th><td>When the pipeline template was created</td></tr>
+  <tr><th><code>created_by</code></th><td>User who created the pipeline template</td></tr>
+  <tr><th><code>updated_at</code></th><td>When the pipeline template was created</td></tr>
+  <tr><th><code>updated_by</code></th><td>User who last updated the pipeline template</td></tr>
+</tbody>
+</table>
+
+## List pipeline templates
+
+Returns a list of an organization's pipeline templates.
+
+```bash
+curl "https://api.buildkite.com/v2/organizations/{org.slug}/pipeline-templates"
+```
+
+```json
+[
+  {
+    "uuid": "018a86cc-db73-7d15-8c68-5023cf8d64c3",
+    "graphql_id": "UGlwZWxpbmVUZW1wbGF0ZS0tLTAxOGE4NmNjLWRiNzMtN2QxNS04YzY4LTUwMjNjZjhkNjRjMw==",
+    "name": "Build template",
+    "description": "Shared build steps configuration",
+    "configuration": "steps:\n  - label: \":hammer: Build\"\n    command: \"scripts/build.sh\"",
+    "available": false,
+    "url": "http://api.buildkite.localhost/v2/organizations/buildkite/pipeline-templates/018a86cc-db73-7d15-8c68-5023cf8d64c3",
+    "web_url": "http://buildkite.localhost/organizations/buildkite/pipeline-templates/018a86cc-db73-7d15-8c68-5023cf8d64c3",
```

Comment:
I note that [our examples for things like Clusters](https://buildkite.com/docs/apis/rest-api/clusters) use production URLs rather than local ones, we should do the same here.

Reply:
Edited the urls in this commit ([e7d11cb](https://github.com/buildkite/docs/pull/2479/commits/e7d11cb314de0cdb038a225b783d28bc4dc7e420))